packages/core/src/components/dropdown/dropdown.stories.tsx

```diff
@@ -294,15 +294,19 @@ const Template = ({
               îles Åland
             </tds-dropdown-option>
         </tds-dropdown>
+        <button onClick="resetDropdown()">Reset Dropdown</button>
     </div>
 
     <script>
-      dropdown = document.querySelector('tds-dropdown')
+      const dropdown = document.querySelector('tds-dropdown');
```

Comment:
```suggestion
      let dropdown = document.querySelector('tds-dropdown');
```

Just so storybook can reassign it when rerendering